.github/workflows/release.yml

```diff
@@ -5,39 +5,38 @@ on:
     tags: ["v[0-9]+.[0-9]+.[0-9]+*"]
 
 jobs:
-  check:
-    name: Check
+  build:
+    name: Build
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        arch: [ "x86_64", "aarch64" ]
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
-      - name: Setup Rust toolchain and cache
-        uses: actions-rust-lang/setup-rust-toolchain@v1.3.7
-      - name: Install just
-        uses: taiki-e/install-action@just
+      - name: Expose git commit data
+        uses: rlespinasse/git-commit-data-action@v1
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        env:
+          RUST_CACHE_KEY_OS: rust-cache-${{ matrix.arch }}-${{ env.GIT_COMMIT_SHORT_SHA }}
+        with:
+          rustflags: '' #Disable.  By default this action sets environment variable is set to -D warnings.  We manage this in the Makefile
+      - uses: taiki-e/install-action@just
       - name: Install requirements
-        run: sudo env PATH=$PATH just ci-prepare
-      - name: Build
-        run: just youki-release
+        if: ${{ matrix.arch == 'x86_64' }}
+        run: | 
+          sudo env PATH=$PATH arch=${{ matrix.arch }} just ci-prepare
+      - name: Install requirements for aarch64
+        if: ${{ matrix.arch == 'aarch64' }}
+        run: cargo install cross --git https://github.com/cross-rs/cross
+      - name: Release build
+        run: arch=${{ matrix.arch }} just youki-release
       - name: test
         # TODO(utam0k): The feature test needs nightly
         # run: just test-basic featuretest test-oci
+        if: ${{ matrix.arch == 'x86_64' }}
         run: just test-basic test-oci
```

Comment:
**note:** Not for this PR, but leaving it as a comment. This is running on ubuntu-20.04, which runs with cgroups-v1. Do we need to test with a newer ubuntu with cgroups-v2? @utam0k

Reply:
Yes, we should. This is a historical liability: runtime-tools did not support cgroup v2, so we used cgroup v1.